pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>gooddata-java</artifactId>
-    <version>2.22.1-SNAPSHOT</version>
+    <version>2.22.2-BB-792-SNAPSHOT</version>
```

Comment:
Plz remove this from PR

Reply:
I have the PR only for the review. Can you please add the Do not merge label or is it not supported in the repo? I added a note to the title.